sycl/include/CL/sycl/INTEL/esimd/esimd_memory.hpp

```diff
@@ -643,6 +643,198 @@ media_block_store(AccessorTy acc, unsigned x, unsigned y, simd<T, m * n> vals) {
 SYCL_EXTERNAL void slm_init(uint32_t size) {}
 
 #endif
+
+template <typename AccessorTy>
+ESIMD_INLINE ESIMD_NODEBUG uint32_t esimd_get_value(AccessorTy acc) {
+#if defined(__SYCL_DEVICE_ONLY__) && defined(__SYCL_EXPLICIT_SIMD__)
+  return __esimd_get_value(AccessorPrivateProxy::getNativeImageObj(acc));
+#else
+  return __esimd_get_value(acc);
+#endif // __SYCL_DEVICE_ONLY__ && __SYCL_EXPLICIT_SIMD__
+}
+
+/// \brief Raw sends load.
+///
+/// @msgDst the destination operand of send message.
+///
+/// @msgSrc0 the first source operand of send message.
+///
+/// @msgSrc1 the second source operand of send message.
+///
+/// @exDesc the extended message descriptor.
+///
+/// @msgDesc the message descriptor.
+///
+/// @execSize the execution size, which must be a compile time constant.
+///
+/// @sfid the shared function ID, which must be a compile time constant.
+///
+/// @numSrc0 the number of GRFs for source-0, which must be a compile time
+/// constant.
+///
+/// @numSrc1 the number of GRFs for source-1, which must be a compile time
+/// constant.
+///
+/// @numDst the number of GRFs for destination, which must be a compile time
+/// constant.
+///
+/// @isEOT 	the flag that indicates whether this is an EOT message, which
+/// must be a compile time constant (optional - default to 0).
+///
+/// @isSendc the flag that indicates whether sendc should be used, which must be
+/// a compile time constant (optional - default to 0).
+///
+/// @mask the predicate to specify enabled channels (optional - default to on).
+///
```

Comment:
Please add info what this operation returns

Reply:
Added comment.